doc/design/OrionHDFSSink.md

```diff
@@ -203,21 +221,10 @@ A configuration example could be:
 
 [Top](#top)
 
-##<a name="section2.1"></a>Important note regarding the binary backend
-Current implementation of the HDFS binary backend does not support any authentication mechanism.
-
-A desirable authentication method would be OAuth2, since it is the standard in FIWARE, but this is not currenty supported by the remote RPC server the binary backend accesses.
-
-Valid authentication mechanims are Kerberos and Hadoop Delegation Token, nevertheless none has been used and the backend simply requires a username (the one configured in `hdfs_username`) in order the `cygnus` user (the one running Cygnus) impersonates it.
-
-Thus, it is not recommended to use this backend in multi-user environment, or at least not without accepting the risk any user may impersonate any other one by simply specifying his/her username.
-
-There exists an [issue](https://github.com/telefonicaid/fiware-cosmos/issues/111) about adding OAuth2 support to the Hadoop RPC mechanism, in the context of the [`fiware-cosmos`](https://github.com/telefonicaid/fiware-cosmos) project.
-
-[Top](#top)
-
 ##<a name="section3"></a>Use cases
-Use `OrionHDFSSink` if you are looking for a JSON or CSV-based document storage growing in the mid-long term.
+Use `OrionHDFSSink` if you are looking for a JSON or CSV-based document storage growing in the mid-long-term in estimated sizes of terabytes for future trending discovery, along the time persistent patterns of behaviour and so on.
+
+For a short-term historic, those required by dashboards and charting user interfaces, other backends are more suited such as MongoDB or MySQL (Cygnus provides sinks for them, as well).
```

Comment:
Maybe FIWARE Comet should be refered from Cygnus documentation? Not only in this specific point, but in every pooint in which short-term historic or STH feature is mentioned...